### ISSUE TITLE
converting target labels to soft labels 

### ISSUE THREAD

**Issue by bigpigboomboomboom** [as791/Adversarial-Example-Attack-and-Defense#2]
the code :
#converting target labels to soft labels
for data in train_loader:
input, label = data[0].to(device),data[1].to(device)
softlabel = F.log_softmax(modelF(input),dim=1)
data[1] = softlabel
it seem does not convert target labels to soft labels , the label doesn't change in the code that follows
